### DISCUSSION TITLE
Downloading audio files to the application

### DISCUSSION THREAD

**Discussion by i-mage** [quasarframework/quasar#9402] [General - Components / Directives / etc] [answered]
I would like to ask you whether it's possible to download mp3's for your application from external site. I would like to keep the initial size of the app as small as possible and then in case user wants to access some sections, audio files would be loaded and stored on the device.

**Comment by ontwikkelfabriek** [comment, marked as answer]
@i-mage
> I would like to ask you whether it's possible to download mp3's for your application from external site.

Yes you can. Storing files on a client's device would only be possible in Cordova/Capacitor mode.

**Reply by i-mage** [reply]
Thank you. So basically it's possible to store the downloaded files and use them in the application (iOS / Android).

**Reply by ontwikkelfabriek** [reply]
> Thank you. So basically it's possible to store the downloaded files and use them in the application (iOS / Android).

Yes, for Cordova you'll need the file plugin:
https://cordova.apache.org/docs/en/10.x/reference/cordova-plugin-file/

And for Capacitor you'll probably need:
https://capacitorjs.com/docs/apis/filesystem

**Reply by i-mage** [reply]
Thanks!
